### ISSUE TITLE
A new customer cannot get his tenant ID by azure cli

### ISSUE THREAD

**Issue by AbelHu** [Azure/azure-xplat-cli#1948]
A new customer is trying azure cli. He just signed up. Then created new subscription "pay as you go" and then got to the portal. Installed azure cli in his machine. Typed below commands and found his tenant id is undefined. 
 
~$ azure -v
0.9.6

~$ azure login
……

~$ azure account list
info:    Executing command account list
data:    Name           Id                                    Tenant Id  Current
data:    -------------  ------------------------------------  ---------  -------
data:    Pay-As-You-Go  xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx  undefined  true
info:    account list command OK

**Comment by devigned** [comment]
Would you run -vv in the login command? We need a little more information.

**Comment by amarzavery** [comment]
@AbelHu - Azure CLI does not support live id authentication. You need a work id or organization id to authenticate. You can create one by creating an Active Directory and then create a user under the active directory. You will have to link the user with subscription to get the ball rolling. Hope this helps.

**Comment by AbelHu** [comment]
Thanks for your help. 
Finally the customer referenced the second video “Building and Accelerating DevOps Pipeline on Azure” by this link[https://www.microsoftvirtualacademy.com/en-US/training-courses/building-linux-based-solutions-on-azure-11333]. Start to watch it from the 14th minute.

Steps:
If you do not currently have a work or school account, and are using a personal account to log in to your Azure subscription, you can easily create one using the following steps.
1.	Login to the Azure Portal, and select Active Directory.
2.	If no directory exists, select Create your directory and provide the requested information.
3.	Select your directory and add a new user. This new user is a work or school account.
During the creation of the user, you will be supplied with both an e-mail address for the user and a temporary password. Save this information as it is needed later.
4.	From the Azure portal, select Settings and then select Administrators. Select Add, and add the new user as a co-administrator. This allows the work or school account to manage your Azure subscription.
5.	Finally, log out of the Azure portal and then log back in using the new work or school account. If this is the first time logging in with this account, you will be prompted to change the password.
6.	Make sure you see your subscriptions when you log in as the work or school account.
For more information on work or school accounts, see Sign up for Microsoft Azure as an Organization.
Link: https://azure.microsoft.com/en-us/documentation/articles/xplat-cli-connect/

**Comment by SyntaxC4** [comment]
I followed these steps and things still didn't work. Luckily I was able to chat with @devigned and work ou the issue I was having which I am leaving here to unblock anyone else that runs into this issue.

It seems that if you have imported your accounts using `azure account download` before adding an organizational account as mentioned in the [steps](https://github.com/Azure/azure-xplat-cli/issues/1948#issuecomment-125478039) above, then you add a the MSA account to your Azure AD to use `azure login` it doesn't do a full sync of the data.

To fix this issue run `azure account clear` to clear all the accounts (I know, unfortunate, but well worth it to be able to run AD related commands) then use `azure login` to import the subscriptions. once you have done this run `azure account list` and you will notice that the tenantID column is filled and you will be able to execute AD related commands.
